database/generate_dummy_data.py

```diff
@@ -70,83 +63,81 @@ def generate_fake_subscribers(fake: Faker, num: int) -> list[tuple]:
     subscribers = []
 
     for i in range(1, num+1):
-        sub_id = i
         email = fake.unique.email()
         first_name = fake.first_name()
         last_name = fake.last_name()
-        subscribers.append((sub_id, email, first_name, last_name))
+        subscribers.append((email, first_name, last_name))
     return subscribers
 
 
 def generate_fake_articles(
         faker: Faker, num_articles: int, source_id: int,
         source_headlines: list
-        ) -> list[tuple]:
+) -> list[tuple]:
     '''Generates fake article table entries.'''
 
     rows = []
 
     for i in range(1, min(len(source_headlines), num_articles) + 1):
-        article_id = i
         article_title = source_headlines[i-1]
         polarity_score = random.triangular(-1.0, 0.0, -0.5)
         date_published = faker.date_time_between(
             start_date='-5d')
         article_url = faker.url()
-        
-        rows.append((article_id, article_title, polarity_score,
-                        source_id, date_published, article_url))
+
+        rows.append((article_title, polarity_score,
+                     source_id, date_published, article_url))
     return rows
 
+
 def generate_article_topic_assignment(headlines: dict) -> list[tuple]:
     '''Generates fake article_topic_assignment table rows.
     '''
 
     assignments = []
-    
+
     for article_id, (_, all_topics) in enumerate(headlines.items(), start=1):
-    
+
         for topic in all_topics:
-    
-            topic_id = TOPICS.index(topic) + 1 
-    
-            assignments.append((len(assignments) + 1, topic_id, article_id))
-    
+
+            topic_id = TOPICS.index(topic) + 1
```

Comment:
Personal preference but I think we can remove the white lines between line 99 and 105 and the function still be clear to read.